ebs_cleanup/app.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+import json
+import logging
+
+import boto3
+
+
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.INFO)
+
+
+def local_volume_filter(volumes):
+    output = []
+
+    for v in volumes:
+        # double-check for attachments
+        if v.attachments:
+            LOG.error(f"Volume {v.volume_id} has attachments: {v.attachments}")
+            continue
+
+        # check the age is greater than 5 minutes
+        tz = v.create_time.tzinfo
+        if datetime.datetime.now(tz) - v.create_time < datetime.timedelta(minutes=5):
```

Comment:
this was primarily intended to avoid race conditions during volume creation/attachment, but yeah I think it could make sense to accept a parameter for the age